framework/src/main/java/org/checkerframework/framework/util/element/TargetedElementAnnotationApplier.java

```diff
@@ -134,8 +134,8 @@ protected void handleInvalid(List<Attribute.TypeCompound> invalid) {
                 remainingInfo.add(r.toString() + " (" + r.position + ")");
             }
             msg.add(remainingInfo.toString());
-            msg.add("Targeted annotations: " + PluginUtil.join(", ", annotatedTargets()));
-            msg.add("Valid annotations: " + PluginUtil.join(", ", validTargets()));
+            msg.add("Targeted annotations: " + SystemUtil.join(", ", annotatedTargets()));
```

Comment:
Is there a reason not to use `String.join` for these? There is a `join` version with arrays and iterables, so aren't all of the `SystemUtil.join` uses replaceable?

Reply:
No, String#join takes an iterable of CharSequences only.  `annotatedTargets()`, for example, is `TargetType[]`.